daemon/src/engine/client/cg_api.h

```diff
@@ -213,6 +215,8 @@ void            trap_Key_SetBinding( int keyNum, int team, const char *cmd );
 void            trap_Key_ClearCmdButtons( void );
 void            trap_Key_ClearStates( void );
 std::vector<int> trap_Key_KeysDown( const std::vector<int>& keys );
+bool            trap_Mouse_GetCursorMode();
```

Comment:
In that case, true/false is a little opaque. Can you switch to an int with descriptive constant names? ie, MOUSE_MOUSE_DELTA, MOUSE_MODE_ABSOLUTE, since the what "mode" is not clear from the function name or arguments.